dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -30,9 +32,22 @@ - (void)openURLInSafari:(NSURL *)URL;
 @implementation DMPlayerViewController
 
 - (void)dealloc {
-  UIWebView *webView = (UIWebView *)self.view;
-  webView.delegate = nil;
```

Comment:
@JanGorman but wait. `self.webView = nil` calls setter which removes view from superview and it's delegate. If you want, I can duplicate this:
```
        _webView.delegate = nil;
        [_webView removeFromSuperview];
        [_webView stopLoading];
```
in dealloc or create separate method for this.